packages/fast-components/src/slider/slider.stories.ts

```diff
@@ -0,0 +1,14 @@
+import { FASTSlider } from ".";
+import { FASTDesignSystemProvider } from "../design-system-provider";
+import Examples from "./fixtures/base.html";
+import { STORY_RENDERED } from "@storybook/core-events";
+import addons from "@storybook/addons";
+
+// Prevent tree-shaking
+FASTSlider;
+FASTDesignSystemProvider;
+
+export default {
+    title: "Slider",
+};
+export const Base = () => Examples;
```

Comment:
Is `Base` a safe const here to be exporting?

Reply:
exporting with an appropriate name now